apps/crunchy-postgres/workload/run_litmus_test.yml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cruncy-postgres-loadgen 
```

Comment:
@shashank855 can you change the typo cruncy -> crunchy

Reply:
sure i will make the changes